### ISSUE TITLE
2021-4-25

### ISSUE THREAD

**Issue by ssfc** [ssfc/wangdao#524]
191feat: The node features.
192Create repository gdl-tutorial; done; separate dictionary content; separate feat; done; separate label; done; separate train_mask; done; separate val_mask; done; (2021-3-30)
193Try to get size of the tensor; row of feat is number of nodes, column of feat is number of features; (2021-3-30)
194Reserve a room; done; (2021-3-30)
195Solve leetcode 27; (2021-3-30)
196Defining a Graph Convolutional Network (GCN); To build a multi-layer GCN you can simply stack dgl.nn.GraphConv modules, which inherit torch.nn.Module; 
197Separate GCN into 4 steps; done; move compute loss out; done; (2021-3-30)
198What is pytorch argmax function? Returns the indices of the maximum value of all elements in the input tensor. (2021-3-30)
199Try to train this on GPU; done; try another way to send to gpu; done; (2021-3-30)
